src/Cameras/Inputs/babylon.flyCameraMouseInput.ts

```diff
@@ -0,0 +1,291 @@
+module BABYLON {
+    /**
+     * Listen to mouse events to control the camera.
+     * @see http://doc.babylonjs.com/how_to/customizing_camera_inputs
+     */
+    export class FlyCameraMouseInput implements ICameraInput<FlyCamera> {

```

Comment:
what about touch (for mibles). I guess it should work for touch as well no?

Reply:
It should, ideally. I'd like to make sure it's better implemented per the original intent with the Mouse, first. <Works on failed checks...>

Reply:
It's comparing the "what's new.md" file against the /dist/ file in my fork? Instead of the file in the main folder, as I expected.

From __gulpfile.js__:
```
// Compare what's new with the current one in the preview release folder.
const https = require("https");
const url = "https://rawgit.com/BabylonJS/Babylon.js/master/dist/preview%20release/what's%20new.md";
https.get(url, res => {
    res.setEncoding("utf8");
    let oldData = "";
    res.on("data", data => {
        oldData += data;
    });
    res.on("end", () => {
        fs.readFile("../../dist/preview release/what's new.md", "utf-8", function(err, newData) {
            if (err || oldData != newData) {
                done();
                return;
            }

            console.error("What's new file did not change.");
            process.exit(1);
        });
    });
});
```

Reply:
in dist/preview release/